src/main/webapp/i18n/en/exam.json

```diff
@@ -44,7 +44,7 @@
             "finishExam": "Finish exam",
             "progress": "Exercise {{ current }} of {{ all }}",
             "progressSmall": "{{ current }}/{{ all }}",
-            "noExam": "Exam could not be loaded. Please contact your instructor.",
+            "noExam": "Exam could not be loaded. It seems you are not registered for the exam. Please contact your instructor.",
```

Comment:
I am wondering if this should be more direct?

> You are not registered for "EXAM NAME HERE". Please contact your instructor.

Reply:
The problem is that we do not have the exam name as this error appears when student exam is undefined. This is also why I wrote 'it seems that'. This could also be due to the exam not being visible for example

Reply:
Maybe you could include the examId. That should be available. Then there is at least some information :)

Reply:
You have the exam name, it is part of the exam object, that should be available here, because the student was able to click into it (it was loaded before in the courses for dashboard call)

Reply:
We cannot fetch the exam if the user does not have access to it / is registered on the server. See ExamResource